opencog/scm/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ ADD_GUILE_MODULE (FILES
 
 ADD_GUILE_MODULE (FILES
 	opencog/exec.scm
+	MODULE_DESTINATION "${GUILE_SITE_DIR}/opencog"
+)
+
+ADD_GUILE_MODULE (FILES
+
```

Comment:
@ferrouswheel why is the split required?

Reply:
It shouldn't be required. Must be a remnant of when I was experimenting. Good catch!